course_info/icommons.py

```diff
@@ -92,6 +108,31 @@ def _get_schools(self, school_id=None, **kwargs):
             raise ICommonsApiValidationError(str(e))
         return rv
 
+    def _course_info_instructor_list(self, course_instance_id=None, **kwargs):
+
+        # use kwargs to append 'people' to the resource url and to get course people data
+        kwargs['append_to_path'] = 'people'
+
+        instructors = None
+        rv = self._get_resource('course_instances', course_instance_id, **kwargs)
+        try:
+            if 'results' in rv:
+                instructors = []
+                for instance in rv['results']:
+                    course_person_schema(instance)
+                    if instance.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
+                        instructors.append(instance)
+            else:
+                course_person_schema(rv)
```

Comment:
Is this resource ever going to return a single item, rather than a list of items?  If not, we don't need this logic.